### PR TITLE
Swagger generated APIs from master

### PR DESCRIPTION
**Pull request by Ropes**
Obviously there have been some changes to code and swagger declarations
which weren't built.

Commands run: `make generate-api` & `make generate-health-api` & `git
add api*`. That's it.

### Backstory 

These changes were noticed when making a modification to the `server.gotmpl` file for #10116. Thus creating a requirement to separate API master code, from what was being changed for the `syscall` package removal.

@aanm asked me to build all of the swagger generated code from master so the diff between #10158 was limited to only the `syscall`->`unix` package changes.

Related: #10116

Signed-off-by: Joshua Roppo joshroppo@gmail.com

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/10336)

### CODE REVIEW

File: api/v1/health/server/server.go [hunk 1]
```diff
@@ -25,8 +22,11 @@ import (
 	flags "github.com/jessevdk/go-flags"
 	"golang.org/x/net/netutil"
 
+	"github.com/cilium/cilium/api/v1/health/server"
```
Comment (joestringer) [inline api/v1/health/server/server.go]: I'm surprised at these changes in `api/v1/health/server/server.go` and `api/v1/server/server.go`, only the imports are changing but there's no other changes. Line 290 for instance uses the `x509` package, so this seems like a breaking change. I think we should drop the changes to these two files?

**Review by joestringer — CHANGES_REQUESTED** [review]
Thanks for preparing this! At first glance this seems good, just one comment for the last couple of files below.

File: api/v1/health/server/server.go (hunk 1 above)
Comment (Ropes) [reply api/v1/health/server/server.go]: Yeah that's a problem.. but we don't even make it that far:
```
make[1]: Leaving directory '/home/josh/go/src/github.com/cilium/cilium/cilium'
make[1]: Entering directory '/home/josh/go/src/github.com/cilium/cilium/daemon'
CGO_ENABLED=0 go build -mod=vendor -ldflags '-X "github.com/cilium/cilium/pkg/version.Version=1.7.90 f411adcbf 2020-02-25T10:34:54-08:00 go version go1.13.6 linux/amd64" -X "github.com/cilium/cilium/pkg/envoy.RequiredEnvoyVersionSHA=c31482c3e49670980c05cafc914320f7949b266f" -X "github.com/cilium/cilium/pkg/datapath/loader.DatapathSHA=5e3b6c150445abcefcc48bd0358fe1b421bb369a"'  -o cilium-agent
import cycle not allowed
package github.com/cilium/cilium/daemon
	imports github.com/cilium/cilium/cilium-health/launch
	imports github.com/cilium/cilium/pkg/health/server
	imports github.com/cilium/cilium/api/v1/health/server
	imports github.com/cilium/cilium/api/v1/health/server
import cycle not allowed
package github.com/cilium/cilium/daemon
	imports github.com/cilium/cilium/api/v1/server
	imports github.com/cilium/cilium/api/v1/server
Makefile:8: recipe for target 'cilium-agent' failed
```

Removing two more files... :grimacing:

**Comment by Ropes** [comment]
Alright.... `make build` passing(at least).

**Review by aanm — APPROVED** [review]
What can we do to not miss these ones? :thinking:

**Comment by aanm** [comment]
test-me-please

**Comment by joestringer** [comment]
test-me-please

**Comment by Ropes** [comment]
@aanm the failure of swagger to produce breaking changes could be caught by running the swagger generation, and then running `make build`. Maybe `generate-k8s-api` too? The underlying cause of the import issue(which still exists) would need to be immediately fixed in that case.

Running the swagger generation on every PR would expose the issues which my first PR uncovered. I'll re-list the issues uncovered in this yak shaving exercise:
* [ ] Swagger incorrect generation of `api/v1/health/models/health_response.go` (Sounds like this issue is already being tracked)
* [ ] swagger removes `// +k8s:deepcopy-gen=true` which breaks downstream consumer.(`make generate-k8s-api` IIUC)
* [ ] Swagger injects invalid/breaking import statements into: api/v1/health/server/server.go & api/v1/server/server.go Cause unknown.

Those are all the issues I know of at this point.

**Review by joestringer — APPROVED** [review]

**Pull request merged**